Move mediatek.passes to mediatek._passes

tarun292 opened the pull request:
Differential Revision: D63926846

cccclai commented:
@neuropilot-captain can you take a look at this?

tarun292 committed:
Move vulkan.passes to vulkan._passes

Differential Revision: D63926849

tarun292 committed:
Move qualcomm.passes to qualcomm._passes

Differential Revision: D63926845

tarun292 committed:
Move cadence.aot.passes to cadence.aot._passes

Differential Revision: D63926847

neuropilot-captain commented:
Should update the import statement in `backends/mediatek/quantizer/quantizer.py` file as well?

It currently use `from ..passes.decompose_scaled_dot_product_attention import ...`

tarun292 committed:
Move mediatek.passes to mediatek._passes (#5922)

Summary:
Changing mediatek.passes to mediatek._passes to indicate that these passes are not covered under the API stability guarantee.

Pull Request resolved: https://github.com/pytorch/executorch/pull/5922

Reviewed By: helunwencser

Differential Revision: D63926846

tarun292 commented:
@pytorchbot cherry-pick --onto release/0.4 -c critical
